lib/bootloader_pvm.pm

```diff
@@ -184,11 +183,19 @@ sub prepare_pvm_installation {
     }
 
     # For Agama unattended tests, disks will be formatted by default
-    if (!is_upgrade && !get_var('KEEP_DISKS') && !get_var('AGAMA_AUTO')) {
+    if (!is_upgrade && !get_var('KEEP_DISKS')) {
         prepare_disks;
     }
 
-    return if is_agama;
+    if (is_agama) {
+        if (get_var('AGAMA_AUTO')) {
+            my $agama_auto = data_url(get_var('AGAMA_AUTO'));
+            assert_script_run("agama profile import $agama_auto");
+            assert_script_run("agama config show");
+            script_run("agama install &");
+        }
+        return;
+    }
```

Comment:
could you please explain this part? when there is unattended installation the profile is passed in kernel parameter, if this is ipmi-specific it shouldn't happen for normal unattended installation or I'm missing something....

Reply:
Thanks for your input, I will try to test :
1. Don't define agama.auto parameters in kernel cmdline, then we can format the disks at the early stage
2. load the agama.auto config file and start the auto installation later 

But, it is just a test, I don't know if it is a good method....

Reply:
And thanks @lemon-suse 's suggestion, I need a new branch to run my test code. let me close this PR for now and   will create a new one if everything is clear

Reply:
installation should be unnattended we cannot test in this way, this is something we use to Arrange the state of a test to not repeat steps but shouldn't be done, otherwise you are testing always interactive installation and running some CLI commands and that is not what customer expect, customer expect a fully unattended process, that is what is sold :)

Reply:
I can't agree with you anymore! let me stop this useless test